rootfs/opt/kubectl/start.sh

```diff
@@ -89,11 +89,13 @@ cat << EOF > "$INI_FILE"
 gke: $INI_GKE
 storage_size: $STORAGE_SIZE
 mariadb_image: $MARIADB_IMAGE
-mem_request: $MEM_REQUEST
+czar_db_mem_request: $CZAR_DB_MEM_REQUEST
 qserv_image: $QSERV_IMAGE
 minikube: $INI_MINIKUBE
```

Comment:
Since minikube is gone, do we need this line?

Reply:
:+1: